src/library/commissioner_impl.cpp

```diff
@@ -1156,7 +1157,7 @@ void CommissionerImpl::SendPetition(PetitionHandler aHandler)
 
     mBrClient.SendRequest(request, onResponse);
 
-    LOG_DEBUG("sent petition request");
+    LOG_DEBUG(LOG_REGION_GENERIC, "sent petition request");
```

Comment:
Prefer to avoid the "GENERIC" name - it is too easy for this to be a catch all for any new logs. Can we use a more specific name here? How is this different than MESHCOP?

Reply:
oh, it seems I misunderstood the scope of MeshCoP and we should use `LOG_REGION_MESHCOP` instead.

Reply:
done.